DigitalLearningSolutions.Web/ViewModels/LearningMenu/InitialMenuViewModel.cs

```diff
@@ -11,6 +12,8 @@ public class InitialMenuViewModel
         public string AverageDuration { get; }
         public string CentreName { get; }
         public string? BannerText { get; }
+        public bool IncludeCertification { get; }

```

Comment:
I'd name this something more descriptive of how we're using it, e.g. `ShouldShowCompletionSummary` 🙂

Reply:
Should I change this in the CourseContent class as well?

Reply:
No. In the CourseContent class the fields should reflect what they represent for the content. This boolean represents for the content that it includes certification. It's only in this class that we're taking that to mean that we should show completion summary. That's specific to this front end class, not to the whole course content